server.go

```diff
@@ -243,6 +245,42 @@ func (s *Server) register(sd *ServiceDesc, ss interface{}) {
 	s.m[sd.ServiceName] = srv
 }
 
+// Metadata returns the metadata for a given symbol name.
+// The name can be a service name or a method name in the form of
+// <package>.<service>[.<method>].
+func (s *Server) Metadata(name string) interface{} {
```

Comment:
Metadata is a receiver function of the Server type. Adding “Server” to the beginning of the name is likely to only cause studdering:

e.g.:
server := GetServer()
meta := server.ServerMetadata() // bad form